Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -317,14 +325,15 @@
     slots:
       gun_magazine:
         name: Magazine
-        startingItem: MagazinePistolHighCapacity
+        startingItem: MagazineScorpion
         insertSound: /Audio/_Sunrise/Weapons/Guns/SMGs/scorpion/scorpion_reload.ogg
         ejectSound: /Audio/_Sunrise/Weapons/Guns/SMGs/scorpion/scorpion_unload.ogg
         priority: 2
         whitelist:
           tags:
           - MagazinePistolSubMachineGun
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 12 but found 10

(indentation)